Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -272,10 +272,16 @@ public static RHG.PolyCurve ToRhino(this BHG.PolyCurve bPolyCurve)
             if (bPolyCurve == null)
                 return null;
 
+            //Curve is checked that it is a joined curve and only converted if that is the case.
```

Comment:
Can we add the date for which this comment becomes valid - beneficial in case we ever want to look back if a future issue arises and work out how we got to this solution.

Reply:
Date is ofc available here on github through the history, but can add it if you like

Reply:
Have discussed this offline with @IsakNaslundBh and am happy without a date being included following discussion behind reasons.